eks/cluster-loader/configs/cluster-autoscaler/deployment.yaml

```diff
@@ -23,15 +23,6 @@ spec:
             requests:
               cpu: {{.CpuRequest}}
               memory: {{.MemoryRequest}}
-      affinity:
```

Comment:
What's the value in removing this affinity and the toleraiton?

Reply:
Discussed offline. This is fine.

Reply:
toleration not needed if it's not kubemark, and same goes with the affinity